prov/efa/src/efa.h

```diff
@@ -426,6 +431,18 @@ bool efa_ep_support_rnr_retry_modify(struct fid_ep *ep_fid)
 #endif
 }
 
+/**
+ * @brief return whether this endpoint will write error cq entry for RNR
+ *
+ * @param[in]	ep	endpoint
+ */
+static inline
+bool rxr_ep_write_rnr_completion(struct rxr_ep *ep)
+{
+	return (rxr_env.rnr_retry < EFA_RNR_INFINITE_RETRY) &&
```

Comment:
if you're going to have the logic around writing completions here, then we should have some description of why these are the two things we check for.  It's also a very weird function name.  Either this determines if the endpoint will write a CQ entry for an RNR or it writes an rnr completion.  make the naming match.

Reply:
Added comments regarding the check. Also changed the name to `rxr_ep_should_write_rnr_completion`